guardianApi.js

```diff
@@ -0,0 +1,26 @@
+const GuardianApiKey = require('./guardianApiKey');
+
+const guardianKey = new GuardianApiKey;
+
+class GuardianApi {
+  constructor() {
+    this.url = `https://content.guardianapis.com/search?q=latest&query-fields=headline&show-fields=thumbnail,headline,byline&order-by=newest&api-key=${guardianKey.key}`;
+  }
+
+  getHeadlines(callback) {
+    fetch(this.url)
+      .then(response => response.json())
+      .then(data => {
+        console.log(data.response.results),
+        callback(data.response.results) 
+      })
+      .catch((error) => console.log(error));
+  }
+
+  searchHeadlines(searchField, callback) {
```

Comment:
This is not a great use of the instance variable as you are first chaging it, then calling another method that uses it.
Because of that, there's no way you can get headlines again after you've done a search, for example.

You may want to make the getHeadline method take a url as a parameter, or have two separate methods that fetch, each with its own url.